p2p/src/authenticated/discovery/types.rs

```diff
@@ -359,12 +343,6 @@ impl<C: PublicKey> InfoVerifier<C> {
         // for selecting a random subset of peers when there are too many) and allow
         // for duplicates (no need to create an additional set to check this)
         for info in infos {
-            // Check if IP is allowed
```

Comment:
This would've actually led to us continually dropping signed IPs for some peer if two peers had misaligned configs...we should instead accept peer records and store (to ensure we don't ask for again).